solana/solana-ibc/programs/solana-ibc/src/transfer/impls.rs

```diff
@@ -330,10 +327,8 @@ impl IbcStorage<'_, '_> {
             return Err(TokenTransferError::ParseAccountFailure);
         }
 
-        // Splitting the denom because the trace prefix is not stripped during `send_transfer`.
         let denom = coin.denom.to_string();
-        let denom = denom.rsplit_once('/').unwrap_or((&denom, &denom)).1;
-        let escrow = get_escrow_account(port_id, channel_id, denom);
+        let escrow = get_escrow_account(port_id, channel_id, &denom);
```

Comment:
So for the time being should we be using just the base denom here? `coin.denom.base_denom`?

Reply:
yes until we figure out the prefix thing.